test/pylib/suite/base.py

```diff
@@ -311,7 +311,9 @@ def __init__(self, test_no: int, shortname: str, suite) -> None:
         self.suite = suite
         self.allure_dir = self.suite.log_dir / 'allure'
         # Unique file name, which is also readable by human, as filename prefix
-        self.uname = "{}.{}.{}".format(self.suite.name, self.shortname.replace("/", "_"), self.id)
+        self.uname = f"{self.suite.name}.{self.shortname.replace('/', '_')}.{self.id}"
+        if xdist_worker_id := get_xdist_worker_id():
+            self.uname = f"{xdist_worker_id}.{self.uname}"
```

Comment:
Will our Jenkins code be able to recognized these now-renamed files?

Reply:
On line 314 I just switched to f-strings, no changes to file names.

As for adding xdist worker as a prefix, our pipelines use wildcards:

```
testsLogFiles=*.log
pyTestLogFile=test.py.log
junitTestXmlFile=junit.xml
xunitTestXmlFile=*.xunit.xml
boostTestXmlFile=boost.*.xml
```